build failure on musl due to use of glibc internal __bswap_16

**Godot version:** 3.0.6 stable
**OS/device including version:** sabotage linux x86_64 using musl libc
**Issue description:**
```
platform/x11/joypad_linux.cpp: In member function 'void JoypadLinux::open_joypad(const char*)':
platform/x11/joypad_linux.cpp:370:52: error: '__bswap_16' was not declared in this scope
   sprintf(uid, "%04x%04x", __bswap_16(inpid.bustype), 0);
```
maybe this should be added to #20037

--- a/platform/x11/joypad_linux.cpp
+++ b/platform/x11/joypad_linux.cpp
@@ -367,12 +367,12 @@ void JoypadLinux::open_joypad(const char *p_path) {
 		joy.fd = fd;
 		joy.devpath = String(p_path);
 		setup_joypad_properties(joy_num);
-		sprintf(uid, "%04x%04x", __bswap_16(inpid.bustype), 0);
+		sprintf(uid, "%04x%04x", BSWAP16(inpid.bustype), 0);
 		if (inpid.vendor && inpid.product && inpid.version) {
 
-			uint16_t vendor = __bswap_16(inpid.vendor);
-			uint16_t product = __bswap_16(inpid.product);
-			uint16_t version = __bswap_16(inpid.version);
+			uint16_t vendor = BSWAP16(inpid.vendor);
+			uint16_t product = BSWAP16(inpid.product);
+			uint16_t version = BSWAP16(inpid.version);
 
 			sprintf(uid + String(uid).length(), "%04x%04x%04x%04x%04x%04x", vendor, 0, product, 0, version, 0);
 			input->joy_connection_changed(joy_num, true, name, uid);

